packages/wallet/src/W3mFrameStorage.ts

```diff
@@ -16,9 +16,13 @@ export const W3mFrameStorage = {
     return null
   },
 
-  delete(key: string) {
+  delete(key: string, social?: boolean) {
     if (W3mFrameHelpers.isClient) {
-      localStorage.removeItem(`${W3mFrameConstants.STORAGE_KEY}${key}`)
+      if (social) {
+        localStorage.removeItem(key)
```

Comment:
Confused about this one, why there is no prefix like `W3mFrameConstants.STORAGE_KEY` for `social`?

Reply:
Because we need to add that item outside of the wallet package. But we need to remove it when we clear the cache.

Reply:
Is this not set by us and thus it lacks the prefix?

Reply:
We set it oursvelves, but outside of the wallet package